app/elements/routing.html

```diff
@@ -95,6 +95,12 @@
 					page.createTeam();
 					return;
 				}
+				if(qs[i].indexOf('invitedby=') === 0 && qs[i + 1].indexOf('teamname=') === 0) {
```

Comment:
this will crash because `i+1` will be greater than `qs.length`. For this function you can use something like [this](https://github.com/coduno/app/blob/dev/app/elements/routing.html#L204) or you can use [this function](https://github.com/coduno/app/blob/dev/app/elements/routing.html#L247)